ecl2df/common.py

```diff
@@ -643,26 +643,28 @@ def parse_lyrfile(filename: str) -> List[Dict[str, Any]]:
     #formation-names-description-files-_lyr_
 
     The output has the following format:
-
+    ```json
```

Comment:
"json" here is not valid  in sphinx it seems.

Reply:
Ok, I will remove it. What about the quotes?